Lib/slapdtest/_slapdtest.py

```diff
@@ -212,16 +222,44 @@ class SlapdObject(object):
     _start_sleep = 1.5
 
     # create loggers once, multiple calls mess up refleak tests
-    _log = combined_logger('python-ldap-test')
+    _log = None
+
+    def __init__(self,
+                 host=None,
+                 port=None,
+                 log_level=logging.WARN,
+                 openldap_schema_files=None,
+                 database=None,
+                 suffix=None,
+                 root_cn=None,
+                 root_pw=None,
+                 datadir_prefix=None,
+                 ):
+        if SlapdObject._log is None:
+            SlapdObject._log = combined_logger('python-ldap-test', log_level=log_level)
```

Comment:
This will change the logger for *all* SlapdObject instances.
Please use `self._log` instead, only set this if `log_level` is not using the default, and ideally change the logger name to reflect the log_level.